javascript/script.js

```diff
@@ -0,0 +1,90 @@
+const statusDisplay = document.querySelector('#score');
+
+let gameActive = true;
+let currentPlayer = "X";
+let gameState = ["", "", "", "", "", "", "", "", ""];
+const winningMessage = () => `Player ${currentPlayer} has won!`;
```

Comment:
good use of the no-curlies syntax! be careful with it though, it leads me make to syntax errors often